huaweicloud/resource_huaweicloud_cce_node_attach.go

```diff
@@ -309,7 +309,7 @@ func resourceCCENodeAttachV3Create(d *schema.ResourceData, meta interface{}) err
 		return fmtp.Errorf("Error adding HuaweiCloud Node: %s", err)
 	}
 
-	nodeID, err := getResourceIDFromJob(nodeClient, s.JobID, "CreateNode", "InstallNode")
+	nodeID, err := getResourceIDFromJob(nodeClient, s.JobID, "CreateNode", "InstallNode", d.Timeout(schema.TimeoutCreate))
```

Comment:
line width exceeds 120, it is recommended to add line breaks

Reply:
fixed